### DISCUSSION TITLE
Online Demo available to use?

### DISCUSSION THREAD

**Discussion by snaildos** [coqui-ai/TTS#585] [Ideas]
Hey! Is there a available online demo for people to use? If not, I can help run a community based one, I just need a docker container to be made.

**Comment by erogol** [comment]
Hey @snaildos. Thx for your interest. 

We do have a CLI interface and a demo server to run locally, as it is explained in the README.md 

We do not have an official docker image but AFAIK @synesthesiam has created one. 

One question,  what exactly do you mean by `community based one`?

**Reply by snaildos** [reply]
Like, demotts website, where you can create tts online!

**Reply by erogol** [reply]
You can maybe try using the docker image I referred above

**Reply by snaildos** [reply]
So are you interested in a demo website where people can try it out and generate tts samples?

**Reply by saona-raimundo** [reply]
I also think it is a good idea to support some sort of online demo, even if it has to be run locally and takes some time to download a model, even if it has character limits and so on.
